### PR TITLE
chore: update downloads to render icon LESQ-240

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

- Renders download Icon for supplementary resources

Downloads currently don't work as waiting for https://www.notion.so/oaknationalacademy/Connect-OWA-to-2023-content-downloads-server-13e214429a24440fa2dea124dacb5cee 

## Issue(s)

Fixes #LESQ-240

## How to test

1. Go to 
https://deploy-preview-1858--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/geography-secondary-ks4/units/test-unit/lessons/lesson-1-in-test-unit/downloads

3. Click on _______
4. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:

Screenshot 2023-09-01 at 16 06 24

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/node-lib/curriculum-api-2023/queries/downloads/downloads.query.ts [hunk 1]
```diff
@@ -12,9 +12,10 @@ const lessonDownloadsQuery =
     lessonSlug: string;
   }) => {
     const res = await sdk.lessonDownloads(args);
-
     const [downloads] = res.downloads;
 
+    console.log(downloads, "<< RESULT FROM QUERY");
```
Comment (JBR90) [inline src/node-lib/curriculum-api-2023/queries/downloads/downloads.query.ts]: 🔥

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/downloads.tsx [hunk 2]
```diff
@@ -91,6 +91,8 @@ const LessonDownloadsPage: NextPage<LessonDownloadsPageProps> = ({
       mode: "onBlur",
     });
 
+  console.log(downloads, "<< from the downloads page");
```
Comment (JBR90) [inline src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/downloads.tsx]: 🔥

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/downloads.tsx [hunk 3]
```diff
@@ -512,6 +514,7 @@ export const getStaticProps: GetStaticProps<
               lessonSlug,
             });
 
+      console.log(curriculumData, "<< curriculumData");
```
Comment (JBR90) [inline src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/downloads.tsx]: 🔥

**Review by JBR90 — CHANGES_REQUESTED** [review]
Looks good, few logs to remove

File: src/node-lib/curriculum-api-2023/queries/downloads/downloads.query.ts (hunk 1 above)
Comment (k-huggs) [reply src/node-lib/curriculum-api-2023/queries/downloads/downloads.query.ts]: HA removed both !

**Review by mantagen — APPROVED** [review]

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/downloads.tsx (hunk 3 above)
Comment (mantagen) [reply src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/downloads.tsx]: 🧯

**Review by jack-skerrett-oak — APPROVED** [review]
I can't see any regressions, but also with the state of downloads at the minute I am not able to see this work. Hopefully once everything is plugged in, it will work seamlessly!

**Pull request merged**